xml/adm_sudo.xml

```diff
@@ -0,0 +1,611 @@
+    as &exampleuserII;, with the restriction that no arguments are specified.
+    This can be used to assign roles with specific abilities to certain users and groups.
+  </para>
+
+  <sect1 xml:id="sec.adm.sudo.usage">
+    <title>Basic &sudo; usage</title>
+
+    <para>
+      &sudo; is simple to use, yet very powerful.
+    </para>
+
+    <sect2 xml:id="sec.adm.sudo.usage.cmd">
+      <title>Running a single command</title>
+      <para>
+        Logged in as normal user, you can run any command as &rootuser; by prepending it with &sudo;.
+        It will prompt for the root password and, if authenticated sucessfully, run the command as &rootuser;:
+      </para>
+
+      <screen>
+&prompt.user;id -un<co xml:id="co.sudo.usage.id"/>
+tux
+&prompt.user;sudo id -un
+root's password:<co xml:id="co.sudo.usage.pw"/>
+root
+&prompt.user;id -un
+tux<co xml:id="co.sudo.usage.after"/>
+&prompt.user;sudo id -un
+<co xml:id="co.sudo.usage.nopw"/>
+root
+</screen>
+
+      <calloutlist>
+        <callout arearefs="co.sudo.usage.id">
+          <para>The <command>id</command> command prints information about a user, the current one if not specified.</para>
+        </callout>
+        <callout arearefs="co.sudo.usage.pw">
+          <para>The password is not shown during input, neither as clear text nor as bullets.</para>
+        </callout>
+        <callout arearefs="co.sudo.usage.after">
+          <para>Only the single command is run with elevated privileges, those after &sudo;
+          are run as normal user again.</para>
+        </callout>
+        <callout arearefs="co.sudo.usage.nopw">
+          <para>For a limited amount of time you do not need to enter the &rootuser; password again.</para>
+        </callout>
+      </calloutlist>
+
+      <tip>
+        <title>I/O redirection</title>
+        <para>
+          I/O redirection does not directly work that way:
+        </para>
+
+        <screen>
+&prompt.user; sudo echo s &gt; /proc/sysrq-trigger 
+bash: /proc/sysrq-trigger: Permission denied
+&prompt.user; sudo cat &lt; /proc/1/maps
+bash: /proc/1/maps: Permission denied
+</screen>
+
+        <para>
+          Only the <command>echo</command>/<command>cat</command> binary is run with elevated privileges,
+          while the redirection is performed by the user's shell.
+
+          You can either start a shell like in <xref linkend="sec.sudo.shell"/>
+          or use the <command>dd</command> utility instead:
+        </para>
+
+        <screen>
+echo s | sudo dd of=/proc/sysrq-trigger
+sudo dd if=/proc/1/maps | cat <!-- TODO: Better example! -->
+</screen>
+      </tip>
+    </sect2>
+
+    <sect2 xml:id="sec.sudo.shell">
+      <title>Starting a shell</title>
+      <para>
+        Having to prepend &sudo; to every command can be cumbersome.
+        While you can specify a shell as a command: <command>sudo bash</command>,
+        there are two options specifically for this purpose.
```

Comment:
While you could specify a shell as a command (<command>sudo bash</command>), it is recommended to rather use one of the built-in mechanisms to start a shell in <quote>susdo mode</quote>: